doc/whatsnew/fragments/8973.feature

```diff
@@ -0,0 +1,3 @@
+Print package stats when running Pyreverse and add a --verbose flag for debugging output.
```

Comment:
```suggestion
Package stats are now printed when running Pyreverse and a ``--verbose`` flag was added to get the original output with parsed modules. You might need to activate the verbose option if you want to keep the old output.
```
I think this need to be a user action type file too but I can't do that in the web interface sorry.